cmake_templates/PyQgsTest.cmake.in

```diff
@@ -0,0 +1,17 @@
+
+SET(ENV{QGIS_PREFIX_PATH} "@QGIS_PYTEST_PREFIX_PATH@")
```

Comment:
Is there a reason to keep commands uppercase in here?

Reply:
thanks, fixed